core/modules/system/system.install

```diff
@@ -3304,6 +3304,17 @@ function system_update_1082() {
   }
 }
 
+/**
+ * Set the default value for "file_additional_public_schemes" setting.
+ */
+function system_update_1083() {
+  $config = config('system.core');
+  if ($config->get('file_additional_public_schemes') == NULL) {
```

Comment:
Do we also need to do an `update_variable_get` (in the other system module conversion of system.core variables) so we don't loose this for sites upgrading from Drupal 7 tomorrow?

Reply:
Good call. I'll add it to the earlier update that covers the other file system settings.